run-repl.py

```diff
@@ -0,0 +1,33 @@
+import os
+# from dotenv import load_dotenv
+
+# load_dotenv()  # take environment variables from .env.
+
+# Code of your application, which uses environment variables (e.g. from `os.environ` or
+# `os.getenv`) as if they came from the actual environment.
+
+os.system("alias python3=python")
+
+
+def runSetup():
+    def alert(missing="API_ID , API_HASH"):
+        print(
+            f"\nCopy your {missing} and save it into Secrets(Environment variables) Sidebar!\n"
+        )
+
+    api_id = os.getenv("API_ID")
+    if api_id is None:
+        alert()
+        return
```

Comment:
make this check a generic one so that all the `required` env variables are checked before continuing.
```suggestion
    req_env_vars = ["API_ID", "API_HASH", "INDEX_SETTINGS"]
    misssing_envs = False
    for env_var in req_env_vars:
        env_value = os.getenv(env_var )
        if env_value is None:
             misssing_envs = True
             alert(env_var)
    
    if misssing_envs:
        return
```